account-analytics/src/main/java/com.bobocode/AccountAnalytics.java

```diff
@@ -74,7 +87,9 @@ public int getNumOfLettersInFirstAndLastNames() {
      * @return total balance of all accounts
      */
     public BigDecimal calculateTotalBalance() {
-        throw new UnsupportedOperationException("It's your job to implement this method"); // todo
+        return accounts.stream()
+                .map(Account::getBalance)
+                .reduce(BigDecimal.ZERO, BigDecimal::add);
```

Comment:
Method `reduce` is overloaded. So you have two options:

- `Optional<T> reduce(BinaryOperator<T> accumulator);`
- `T reduce(T identity, BinaryOperator<T> accumulator);`

I used the second one, that has an identity (initial) value, **because I wanted to receive real value rather than optional**